distributed/tests/test_scheduler.py

```diff
@@ -1068,6 +1068,30 @@ def test_service_hosts_match_scheduler():
         yield s.close()
 
 
+@pytest.mark.skipif(not sys.platform.startswith('linux'),
+                    reason="Need 127.0.0.* to mean localhost")
+@gen_test(timeout=None)
```

Comment:
Can we remove `timeout=None` to re-enable the standard 10s timeout?

Reply:
I can - it was copy-pasted from the test above it, which is very similar. Should I remove it from both tests?

Reply:
Yes, that sounds good